### ISSUE TITLE
When alt-tabbing to a command prompt, review mode does not work until nvda-num-minus is used.

### ISSUE THREAD

**Issue by eleuteri** [nvaccess/nvda#5791]
When using several applications, and going to a command prompt, review mode won't be able to read it with, for example, num7 or num9. It says "top, blank" or "bottom, blank". When using ins+num-minus, review starts working correctly.

A similar issue seems to occur when tabbing to firefox, when on browse mode on an edit field, but I'm not sure this reliably occurs.

**Comment by Brian1Gaff** [comment]
Which version of windows is this on, by the way?
Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by jcsteh** [comment]
Did you perhaps switch to screen review? Screen review can't work in 
command consoles and Firefox, so you'll only get blank. However, as per 
user demand, once you activate screen review, it "sticks" until deactivated.

**Comment by ehollig** [comment]
@eleuteri, is this still an issue? As @jcsteh suggested in https://github.com/nvaccess/nvda/issues/5791#issuecomment-192878901, try it when NVDA is in object review. When I am in screen review, I get the behavior you describe. However, when in object review, everything works as expected.

**Comment by Adriani90** [comment]
We didn't hear anything from the author for more than 2.5 years. I am closing this issue. If someone reports it again we can reopen it.
